### PR TITLE
Feature/rifex 47/delete rns subdomain

### PR DESCRIPTION
**Pull request by JONAF2103**
This PR introduces the UI for RNS subdomain deletion, we can delete subdomains from the subdomain list or from the subdomain popup. Here is a small demo of this:

demo-11-05-2020-jonathan-delete-subdomain

This affects the UI and the Service layer.

### CODE REVIEW

File: app/scripts/controllers/rif/rns/register/index.js [hunk 1]
```diff
@@ -216,4 +218,31 @@ export default class RnsRegister extends RnsJsDelegate {
     return Promise.resolve(transactionListener.id);
   }
 
+  deleteSubdomain (domainName, subdomain) {
+    // TODO: implement using call contract directly because rnsjs doesn't work for send operations.
+    domainName = this.addRskSuffix(domainName);
+    let subdomains = this.getSubdomains(domainName);
+    if (subdomains.filter(sd => sd.name === subdomain).length > 0) {
+      return new Promise((resolve, reject) => {
+        this.rnsJs.subdomains.setOwner(domainName, subdomain, rns.defaultAddress)
```
Comment (marcosmartinez7) [inline app/scripts/controllers/rif/rns/register/index.js]: This must be implemented with our custom listener right? 

Also, without using rnsjs for web3 complience issues

Comment (JONAF2103) [reply app/scripts/controllers/rif/rns/register/index.js]: Yup we need to use our send method and manage the result using the listener.

**Review by rorolopetegui — APPROVED** [review]
lgtm

**Review by marcosmartinez7 — APPROVED** [review]
lgtm

**Pull request merged**